migrations/v3.js

```diff
@@ -0,0 +1,27 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('Matching - v2.2.0 to v3.0.0', async () => {
+  let matchings, course, courseMatchingGlobals;
+  const originalAriaRegion = 'This question component requires you to select the matching answer from a drop down list below. When you have selected your answers select the submit button.';
+  whereFromPlugin('Matching - from v2.2.0', { name: 'adapt-contrib-matching', version: '<3.0.0' });
+  whereContent('Matching - where matching', async content => {
+    matchings = content.filter(({ _component }) => _component === 'matching');
+    return matchings.length;
+  });
+  mutateContent('Matching - add globals ariaRegion attribute', async content => {
+    course = content.find(({ _type }) => _type === 'course');
+    courseMatchingGlobals = course._globals._components._matching;
```

Comment:
Consider adding a check for existence of globals (see https://github.com/adaptlearning/adapt-contrib-narrative/blob/issue/318/migrations/v4.js#L20)